freshmaker/pyxis.py

```diff
@@ -164,7 +164,13 @@ def get_latest_bundles(self, index_images):
 
             request_params['filter'] = \
                 f'latest_in_channel==true&source_index_container_path=={path}'
```

Comment:
Should be `f'latest_in_channel==true and source_index_container_path=={path}'`, `&` is invalid logical operator in RSQL expression, it's not introduced in your PR, but could you also include a commit to fix this?

Reply:
Good catch. Thank you.